### PR TITLE
Add new htaccess cheat sheet

### PR DESCRIPTION
**Pull request by sahildua2305**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.

- [x] New Instant Answer
    - [x] Cheat Sheets: **`New htaccess Cheat Sheet`**

###### People to notify (@mention interested parties)
@moollaza 

---

Instant Answer Page: https://duck.co/ia/view/htaccess_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by moollaza** [comment]
Restarted the test -- should pass now 👍

**Comment by sahildua2305** [comment]
@moollaza Why was that happening anyway? It has been the case with every new cheat sheet. It would throw the same error initially and then pass later on.

**Comment by moollaza** [comment]
> Why was that happening anyway?

Once we see a PR has been made we move the IA from Planning to Development at which point it's available in the Metadata. After it's available, subsequent tests will pass.

I believe we have a task to add planning IAs to the public metadata to prevent this.

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=htaccess cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/htaccess_cheat_sheet_mobile.png?IMhbZpgv34)

**Comment by sahildua2305** [comment]
@moollaza This is good to go, imo.

**Comment by sahildua2305** [comment]
@moollaza Please review this.

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=htaccess cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/htaccess_cheat_sheet_mobile.png?np5GZBTaRZ)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=htaccess cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/htaccess_cheat_sheet_mobile.png?wUye0tgiRC)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=htaccess cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/htaccess_cheat_sheet_mobile.png?N5TjXUO7ij)

**Comment by sahildua2305** [comment]
Relax @daxtheduck :stuck_out_tongue:

**Comment by moollaza** [comment]
Thanks! This LGTM :+1:

File: share/goodie/cheat_sheets/json/htaccess.json [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+            {
+                "key": "RewriteEngine on\nRewriteCond %\\{HTTP_REFERER\\} site-to-block\\.com \\[NC\\]\nRewriteCond %\\{HTTP_REFERER\\} site-to-block-2\\.com \\[NC\\]\nRewriteRule .* - \\[F\\]",
+                "val": "Block site from specific referrers"
+            },
+            {
+                "key": "RewriteEngine on\nRewriteCond %\\{HTTP_REFERER\\} !^$\nRewriteCond %\\{HTTP_REFERER\\} !^http:\/\/(www\\.)?mydomain.com\/.*$ \\[NC\\]\nRewriteRule \\.(gif|jpg)$ - \\[F\\]",
+                "val": "Block Hot Linking/Bandwidth hogging"
+            },
+            {
+                "key": "# Avoid 500 error by passing chatset\nAddDefaultChatset utf-8",
```
Comment (gasi) [inline share/goodie/cheat_sheets/json/htaccess.json]: Typo: I believe `chatset` and `AddDefaultChatset` should be `charset` and `AddDefaultCharset`.

**Pull request merged**